ignite/manifest.json

```
@@ -0,0 +1,30 @@
+{
+  "display_name": "Ignite",
+  "maintainer": "help@datadoghq.com",
+  "manifest_version": "1.0.0",
+  "name": "ignite",
+  "metric_prefix": "ignite.",
+  "metric_to_check": "",
+  "creates_events": false,
+  "short_description": "Collect metrics from your Ignite server.",
+  "guid": "fd5a21d5-ddfe-4d04-855f-28492b4d270e",
+  "support": "core",
+  "supported_os": [
+    "linux",
+    "mac_os",
+    "windows"
+  ],
+  "public_title": "Datadog-ignite Integration",
+  "categories": [
+    ""
+  ],
```

```suggestion
  "categories": [
    "caching",
    "data store",
  ],
```
some suggested categories